lib/tasks/connectivity.rake

```diff
@@ -0,0 +1,172 @@
+    end
+  end
+
+  desc 'Check GI'
+  task gi: :environment do
+    begin
+      GI::Client.new.get_autocomplete_suggestions('university')
+      puts "GIDS connection success for #{Settings.gids.url}."
+    rescue => e
+      puts "GIDS connection unsuccessful for #{Settings.gids.url}!"
+      puts " - Error encountered: `#{e}`"
+    end
+  end
+
+  desc 'Check HCA'
+  task hca: :environment do
+    begin
+      HCA::Service.new.health_check
+      puts "HCA connection success for #{Settings.hca.endpoint}."
+    rescue => e
+      puts "HCA connection unsuccessful for #{Settings.hca.endpoint}!"
+      puts " - Error encountered: `#{e}`"
+    end
+  end
+
+  desc 'Check that logs are writeable'
+  task logs: :environment do
+    if File.writable?(Rails.root.join("log"))
+      puts "Logging directory is writeable."
+    else
+      puts "Logging directory is not writeable!"
+    end
+  end
+
+  desc 'Check MVI'
+  task mvi: :environment do
+    begin
+      user = User.new(
+        uuid: SecureRandom.uuid,
+        first_name: 'Foo',
+        middle_name: 'B',
+        last_name: 'Fooman',
+        birth_date: '1901-01-01',
+        gender: 'M',
+        ssn: '111221122',
+        email: 'foo@bar.com',
+        loa: {
+          current: LOA::THREE,
+          highest: LOA::THREE
+        }
+      )
+      user.va_profile
+      puts "MVI connection success for #{Settings.mvi.url}."
+    rescue => e
+      puts "MVI connection unsuccessful for #{Settings.mvi.url}!"
+      puts " - Error encountered: `#{e}`"
+    end
+  end
+
+  desc 'Check Redis'
+  task redis: :environment do
+    begin
+      Redis.current.get('asdf')
+      puts "Redis connection success for #{Settings.redis.host}:#{Settings.redis.port}"
+    rescue => e
+      puts "Redis connection unsuccessful for #{Settings.redis.host}:#{Settings.redis.port}"
+      puts " - Error encountered: `#{e}`"
+    end
+  end
+
+  desc 'Check Rx'
+  task rx: :environment do
+    begin
+      Rx::Client.new(session: { user_id: '12210827' }).authenticate
+      puts "Rx connection success for #{Settings.mhv.rx.host}."
+    rescue => e
+      puts "Rx connection unsuccessful for #{Settings.mhv.rx.host}!"
+      puts " - Error encountered: `#{e}`"
+    end
+  end
+
+  desc 'Check SM'
+  task sm: :environment do
+    begin
+      SM::Client.new(session: { user_id: '12210827' }).authenticate
+      puts "SM connection success for #{Settings.mhv.sm.host}."
+    rescue => e
+      puts "SM connection unsuccessful for #{Settings.mhv.sm.host}!"
+      puts " - Error encountered: `#{e}`"
+    end
+  end
+
+  desc 'Check StatsD'
+  task statsd: :environment do
+    if Settings.statsd.host.present? && Settings.statsd.port.present?
+      puts "StatsD configured for #{Settings.statsd.host}:#{Settings.statsd.port}."
+    else
+      puts 'StatsD not configured!'
+    end
+  end
+
+  desc 'Check VHA'
+  task vha: :environment do
+    begin
+      VHAFacilityAdapter.new.query([0, 0, 0, 0])
```

Comment:
This pattern is re-used often and could possibly be made into a method:  
```ruby
def check(code, name, config)
  eval(code)
  puts "#{name} connection success for #{config}"
rescue => e
  puts "#{name}  connection unsuccessful for #{config}"
  puts " - Error encountered: `#{e}`"
end
```

The downside being that `eval()` will run arbitrary code and could be a security concern depending on who has access to run this rake task.